packer/timeboost/overlay/etc/systemd/system/timeboost.service

```diff
@@ -1,15 +1,15 @@
 [Unit]
 Description=Timeboost
 After=network.target nitro.service
-Requires=network-online.target
-Wants=nitro.service
+Requires=network-online.target nitro.service
 
 [Service]
 ExecStart=/usr/local/bin/timeboost -c /etc/timeboost/timeboost.toml
 Environment=RUST_LOG_FORMAT=json
 Environment=RUST_LOG={{LOG_DIRECTIVES}}
 Restart=always
 RestartSec=5
+TimeoutStartSec=infinity
```

Comment:
Why is this needed? Startup should be more or less instant. I would not expect systemd to count the time a dependency requires for its own startup to count against the dependent's startup duration.

Reply:
IIUC `TimeoutStartSec` is only relevant for services that signal startup completion. `timeboost.service` is of type `simple`, i.e. systemd considers timeboost started right after it `fork`ed it. Furthermore, systemd knows that timeboost depends on nitro, but it would not count nitro's startup time against timeboost's.